<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlottingDialog</class>
 <widget class="QDialog" name="PlottingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1763</width>
    <height>1001</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#plottingWidget{
	color: white;
	background-color: rgb(44, 44, 44);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="plottingWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,20,1,1,1">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item alignment="Qt::AlignHCenter">
       <widget class="QWidget" name="dataToShowWidget" native="true">
        <layout class="QHBoxLayout" name="dataToShowLayout">
         <item>
          <widget class="QComboBox" name="selectItemCombo"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QVBoxLayout" name="chartLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="commandsWidget" native="true">
        <layout class="QHBoxLayout" name="commandsLayout"/>
       </widget>
      </item>
      <item alignment="Qt::AlignRight">
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>STOP</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignLeft">
          <widget class="QWidget" name="widget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>10</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>100</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Samples:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="fftSamplesCountCB">
              <property name="currentIndex">
               <number>3</number>
              </property>
              <item>
               <property name="text">
                <string>128</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>256</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>512</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1024</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2048</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>4096</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>8192</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>16384</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>32768</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>65536</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>131072</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>262144</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>524288</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1048576</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>67108864</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="fftBtn">
              <property name="text">
               <string>Generate FFT</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">co</string>
           </property>
           <property name="text">
            <string>XRange</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="XRangeEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>YRange</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="yRangeEdit">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PlottingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PlottingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
